Jenkinsfile

```diff
@@ -12,6 +12,7 @@ pipeline {
         disableConcurrentBuilds()
         timestamps()
         skipDefaultCheckout()
+        disableResume()
```

Comment:
❔ is disabling resuming the build on master restart something we should have in all of of our jobs?

Reply:
Probably. Single use containers aren't really good at starting in the middle of a build.

Reply:
@AdamBurstynski / @shaundmorris great catch, yes we don't want Jenkins trying to resume builds. This is one of the cases where we experience jobs hanging on Jenkins.